refinery/core/test_views.py

```diff
@@ -952,16 +952,27 @@ def test_get_event_list(self):
 
         get_request = self.factory.get(urljoin(self.url_root, '/'))
         get_request.user = self.user
-        get_response = self.view(get_request).render()
-
+        get_response = self.view(get_request)
         self.assertEqual(
-            json.loads(get_response.content),
+            get_response.data,
             [
                 {
-                    'date_time': date_times[0],
-                    'message': messages[0],
+                    'date_time': date_times[3],
+                    'message': messages[3],
                     'data_set': DataSetSerializer(
-                        data_sets[0], context={'request': get_request}
+                        data_sets[3], context={'request': get_request}
+                    ).data,
+                    'group': None,
+                    'user': UserSerializer(self.user).data,
+                    'type': 'UPDATE',
```

Comment:
Originally the type was one of the CRUD verbs, so I'm surprised this isn't `CREATE`?

Reply:
... but it looks like it's just being copied from below, so not a new issue...